content/enterprise/install/automated/active-active.mdx

```diff
@@ -230,6 +228,45 @@ Finally, confirm the application is functioning as expected when running multipl
+
+- [AWS - RDS connection limits](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html#RDS_Limits.MaxConnections)
+- [AWS - Aurora Scaling](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraMySQL.Managing.Performance.html)
+- [Azure - Azure Database Limits](https://docs.microsoft.com/en-us/azure/postgresql/concepts-limits)
+- [Google Cloud - Cloud SQL Quotes and Limits](https://cloud.google.com/sql/docs/quotas)
+- [PostgreSQL 12 - Connection Documentation](https://www.postgresql.org/docs/12/runtime-config-connection.html)
+
+#### Redis Server
+
+Rarely, certain types of workloads may cause spikes in the Redis server CPU or memory. We recommend the Redis server be monitored and scaled up as necessary. 
+
+- [AWS - Monitoring ElastiCache for Redis with CloudWatch](https://aws.amazon.com/blogs/database/monitoring-best-practices-with-amazon-elasticache-for-redis-using-amazon-cloudwatch/)
+- [Azure - Monitor Azure Cache for Redis](https://docs.microsoft.com/en-us/azure/azure-cache-for-redis/cache-how-to-monitor)
+- [Google Cloud - Monitoring Redis Instances](https://cloud.google.com/memorystore/docs/redis/monitoring-instances)
+
+#### Network Infrastructure/API Limits
+
+As you scale out Terraform Enterprise beyond two nodes, you may be adding additional stress to your network and dramatically increasing the number of API calls made in your cloud account. Each of the three support clouds have their own default limits and processes by which those limits might possibly be increased. 
+
+- [AWS - EC2 instance network limits](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-network-bandwidth.html)
+- [AWS - Request Throttling for the EC2 API](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/throttling.html)
+- [Azure - Virtual Machine Network Limits](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-machine-network-throughput)
+- [Azure - Resource Manager Throttling](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/request-limits-and-throttling)
+- [Google Cloud - Network Quotas and Limits](https://cloud.google.com/vpc/docs/quota)
+- [Google Cloud - API rate limits](https://cloud.google.com/compute/docs/api-rate-limits)
+
+Please consult the documentation and support of your cloud platform for more information on any of these topics.
+
+#### Consider Agents
+
+An alternative to scaling Terraform Enterprise beyond two, or even three, nodes is to utilize [Terraform Cloud Agents](https://www.terraform.io/enterprise/admin/agents-on-tfe). When agents are utilized, the actual Terraform plans and applies will take place on the target system that has the agent executable installed, with a much smaller impact on the Terraform Enterprise servers than local runs. Agents can run in other regions, other clouds, and even private cloud, provided the network connectivity is configured to allow for it. 
```

Comment:
```suggestion
Instead of scaling Terraform Enterprise beyond two or three nodes, you can use [Terraform Cloud Agents](https://www.terraform.io/enterprise/admin/agents-on-tfe). Terraform Cloud Agents can run in other regions, other clouds, and even private clouds. Agents poll Terraform Enterprise for work and then Terraform plans and applies will run on the target system that has the agent executable installed. This has a much smaller impact on the Terraform Enterprise servers than running Terraform locally.
```